demo/src/app/components/collapse/demos/default-close/default.close.collapse.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+
+import { NgbCollapse } from '@ng-bootstrap/collapse';
+
+@Component({
+  selector: 'ngbd-collapse-default-close',
+  template: require('./default.close.collapse.html'),
+  directives: [NgbCollapse]
+})
+export class CollapseDefaultCloseComponent {
+  isCollapsed: boolean = true; // Keep content collapsed by default
```

Comment:
should be `private`, don't need data type as it's inferred from the initial value.